### PR TITLE
Initial support for SamKnows SK-WB8

### PR DESCRIPTION
**Pull request by ndoo**
Signed-off-by: Andrew Yong <me@ndoo.sg>

### CODE REVIEW

File: target/linux/ramips/base-files/etc/board.d/01_leds [hunk 1]
```diff
@@ -247,6 +247,9 @@ rt-n14u)
 	set_wifi_led "$board:blue:air"
 	set_usb_led "$board:blue:usb"
 	;;
+sk-wb8)
+	set_usb_led "$board:green:usb"
```
Comment (pepe2k) [inline target/linux/ramips/base-files/etc/board.d/01_leds]: There are other boards (all0239-3g, hw550-3g ...) with same code (set_usb_led "$board:green:usb"), please combine sk-wb8 with them.

**Comment by ndoo** [comment]
Changes made.

**Comment by blogic** [comment]
@pepe2k are you fine with the changes ? you already saw more than me :-) if ok, i would merge this to my local staging tree and fix up the subject/description on the way.

**Comment by pepe2k** [comment]
Looks perfect for me, @ndoo great job!

**Comment by ndoo** [comment]
Cool, thanks for all the help from @blogic & yourself @pepe2k. I have learnt a lot.

**Comment by blogic** [comment]
merged into my staging tree, will be part of source.git when i merge the next time

**Pull request closed without merging**